simulation-scripts/scenario/node-amphibious-operations/apply/scenario.yaml

```diff
@@ -4,14 +4,13 @@ metadata:
   name: node-amphibious-operations
 ---
 apiVersion: apps/v1
-kind: Deployment
+kind: DaemonSet
```

Comment:
This should not be a daemonset. The point is that they may have to try different nodes.